modules/api/php/endpoint.class.inc

```diff
@@ -66,13 +56,6 @@ abstract class Endpoint implements RequestHandlerInterface
             return new \LORIS\Http\Response\JSON\BadRequest('Unsupported version');
         }
 
-        // Hack so that phan doesn't forget it's a RequestHandler inside
-        // the if statement when passed as an argument to process
-        $h = $handler;
-        if ($handler instanceof \LORIS\Middleware\ETagCalculator) {
-            return (new \LORIS\Middleware\ETag())->process($request, $h);
-        }
-
-        return $handler->handle($request);
```

Comment:
@driusan this was part of the initial PR and was part of the conflicts. not sure if your "hack" is still required in the changes (doesnt seem like it) so I removed it